src/app/meds-n-labels/activity-2/activity-2.component.ts

```diff
@@ -36,26 +39,34 @@ export class MemoryGameComponent implements OnInit {
     public alerts: any;
 
     private _faces = [
+                'area-1.png',
                 'area-1.png',
                 'blood-test.png',
-                'enlarged-liver-1.png',
                 'blood-test.png',
+                'enlarged-liver-1.png',
+                'enlarged-liver-1.png',
+                'enlarged-spleen-1.png',
                 'enlarged-spleen-1.png',
                 'other-tests.png',
-                'enlarged-liver-1.png',
-                'rapid-diagnostic-test-1.png',
                 'other-tests.png',
+                'rapid-diagnostic-test-1.png',
+                'rapid-diagnostic-test-1.png',
                 'symptoms-1.png',
                 'symptoms-1.png',
-                'enlarged-spleen-1.png',
-                'area-1.png',
                 'virus-1.png',
-                'rapid-diagnostic-test-1.png',
                 'virus-1.png',
             ];
+    public levelConfig = [{cardsCount: 8, rowCount: 4, columnCount: 2},
+                          {cardsCount: 12, rowCount: 4, columnCount: 3},
+                          {cardsCount: 16, rowCount: 4, columnCount: 4}
+                          ];
+    public currentLevel: any;
+    public  currStage = 7;
```

Comment:
is this changing ? make it constant otherwise.

Reply:
Previous PR.